src/types/annotator.ts

```diff
@@ -349,4 +349,10 @@ export type SideBySideMode = 'auto' | 'manual';
 
 export type SideBySideOptions = {
   mode: SideBySideMode;
+
+  /**
+   * A callback that Hypothesis will call when `mode` is "manual" to determine
+   * whether side-by-side layout has been applied or not.
+   */
+  isActive?: () => boolean;
```

Comment:
Does it make sense to allow providing `isActive` with `mode: 'auto'`? If not, maybe we could define this as a union of `{ mode: 'auto' } | { mode: 'manual'; isActive?: () => boolean }`

Reply:
Yes, that would model the configuration better.